docs-2.0/graph-computing/algorithm-description.md

```diff
@@ -275,11 +392,20 @@ Parameter descriptions are as follows:
 
 - Nebula Analytics
 
-  |Parameter|Predefined value|Description|
-  |:--|:--|:--|
-  |`ITERATIONS`|`10`|The maximum number of iterations.|
-  |`IS_DIRECTED`|`true`|Whether to consider the direction of the edges. If set to `false`, the system automatically adds the reverse edge.|
-  |`IS_CALC_MODULARITY`|`false`| Whether to calculate modularity.|
+  - Input parameters
+
+    |Parameter|Predefined value|Description|
+    |:--|:--|:--|
+    |`ITERATIONS`|`10`|The maximum number of iterations.|
+    |`IS_DIRECTED`|`true`|Whether to consider the direction of the edges. If set to `false`, the system automatically adds the reverse edge.|
+    |`IS_CALC_MODULARITY`|`false`| Whether to calculate modularity.|
+
+  - Output parameters
+
+    |Parameter|Type|Description|
+    |:--|:--|:--|
+    |`VID`|Determined by `vid_type`| The vertex ID.|
+    |`LABEL`|The same type as `VID`| Outputs the vertex IDs that are the same as the label.|
```

Comment:
what do you mean "the same type as VID"   

the same data type as that of the VID?